addons/mail/static/src/views/web/activity/activity_cell.js

```diff
@@ -25,21 +35,22 @@ export class ActivityCell extends Component {
         this.contentRef = useRef("content");
     }
 
-    get closestDeadlineFormatted() {
-        const date = luxon.DateTime.fromISO(this.props.closestDeadline);
-        // To remove year only if current year
-        if (luxon.DateTime.now().year === date.year) {
-            return date.toLocaleString({
-                day: "numeric",
-                month: "short",
-            });
-        } else {
-            return date.toLocaleString({
-                day: "numeric",
-                month: "short",
-                year: "numeric",
-            });
-        }
+    get closestDateFormatted() {
```

Comment:
I remember we discussed about it and I asked some simplification, but after a new year it would be weird actually ... maybe revert the change ?